paymentsource.go

```diff
@@ -36,20 +41,46 @@ func (p *SourceParams) AppendTo(body *form.Values, keyParts []string) {
 // Customer object's payment sources.
 // For more details see https://stripe.com/docs/api#sources
 type CustomerSourceParams struct {
-	Params   `form:"*"`
-	Customer *string       `form:"-"` // Goes in the URL
-	Source   *SourceParams `form:"*"` // SourceParams has custom encoding so brought to top level with "*"
+	Params            `form:"*"`
+	Customer          *string                   `form:"-"` // Included in URL
+	AccountHolderName *string                   `form:"account_holder_name"`
+	AccountHolderType *string                   `form:"account_holder_type"`
+	AddressCity       *string                   `form:"address_city"`
+	AddressCountry    *string                   `form:"address_country"`
+	AddressLine1      *string                   `form:"address_line1"`
+	AddressLine2      *string                   `form:"address_line2"`
+	AddressState      *string                   `form:"address_state"`
+	AddressZip        *string                   `form:"address_zip"`
+	ExpMonth          *string                   `form:"exp_month"`
+	ExpYear           *string                   `form:"exp_year"`
+	Name              *string                   `form:"name"`
+	Owner             *PaymentSourceOwnerParams `form:"owner"`
+	Source            *SourceParams             `form:"*"` // SourceParams has custom encoding so brought to top level with "*"
+}
+type PaymentSourceOwnerParams struct {
+	Address *AddressParams `form:"address"`
+	Email   *string        `form:"email"`
+	Name    *string        `form:"name"`
+	Phone   *string        `form:"phone"`
 }
 
 // SourceVerifyParams are used to verify a customer source
 // For more details see https://stripe.com/docs/guides/ach-beta
 type SourceVerifyParams struct {
 	Params   `form:"*"`
+	Customer *string   `form:"-"`       // Included in URL
```

Comment:
Is this used? The `Verify` method in `paymentsource/client.go` uses a passed-in `id` parameter instead of taking the id off params.

Reply:
@richardm-stripe This line was already there, but it got moved (previously on line 49)

Reply:
Ah, missed that!